registry/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package registry provides types and methods that store
+// information that currently span the whole test run.
+package registry
```

Comment:
I like this!
Considering this. Do you think we should move also the `pidRegistry` implementation to this package?

Reply:
Yeah, happy to do this in another PR 👍